Misc. changes

martinwe-adfinis opened the pull request.

martinwe-adfinis committed:
karuiwm/10-xinput: Return wandering dollar sign to its proper location

Not all who wander are lost.

martinwe-adfinis committed:
karuiwm/10-xinput: Improve device type extraction and code comments

martinwe-adfinis committed:
Add bin/wacom_redimension, for adjusting Wacom table aspect ratio

martinwe-adfinis committed:
sh/aliases: Update fixtgshit to also remove the .desktop file

Since we're now also tracking XDG_DATA_HOME.

martinwe-adfinis committed:
etc/vim: Use separate colour for control sequences in nroff

martinwe-adfinis committed:
etc/systemd/user: Add failing unit

This is useful for testing systemd behaviour when units have failed.

martinwe-adfinis committed:
etc/vim: Add back mediawiki plugin

martinwe-adfinis committed:
etc/vim: Treat .timer.j2 files as systemd unit files

ayekat reviewed (changes requested):
Please…

 * Remove the pipewire "workaround" (+ reversion). It's certainly nice for historic reasons, but I don't think we need this.
 * Remove the `fixdpi` script (we solve this with a proper `Xresource(.d)` entry now; see 20b84839).
 * Remove the ICAClient wrap attempt (as it doesn't work anyway, as stated in the commit message).
 * Rebase the `scrlock` change to the end (I'd like to exclude this one **for now**). I'm thinking about using a wrapper instead (as stated in the commit message).

The rest is fine.

martinwe-adfinis commented:
Dropped the Pipewire back-and-forth, the failed ICAclient wrap attempt and the `fixdpi` script; and moved the `scrlock` change to the end. But I guess in order to merge this (at least in a way that GitHub also considers it "merged", I might need to temporarily split that last commit into a separate branch…
